starter_script.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# The starter script for each VM instance of VMN
+# It installs the VNCServer, sets its password, starts a server, and
+# initiates noVNC's Websocket to TCP proxy/bridge Websockify.
+
+# This URL accesses the metadata value of key "session-id" of the current
+# VM, it will be used as a password to connect to the VNCServer.
```

Comment:
You might want to add 'set -e' here. This causes the script to stop executing if any of the steps fail.

Reply:
set -u and set -x are also common, but I'll let you use your own judgement here.